### PR TITLE
We should consider default fields in relationship constrains

### PR DESCRIPTION
**Pull request by btipling**
as is required for external ticketing which depends on ticketing a default field

## Summary by CodeRabbit

- **New Features**
	- Enhanced configuration handling by ensuring the schema for CLI commands contains unique fields, improving command execution and options presentation.
  
- **Bug Fixes**
	- Resolved potential issues with duplicate field names in the configuration schema.

### CODE REVIEW

**Review by laurenleach — APPROVED** [review]

File: pkg/config/config.go [hunk 1]
```diff
@@ -43,13 +43,24 @@ func DefineConfiguration(
 	v.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 	v.AutomaticEnv()
 
+	confschema := schema
+	confschema.Fields = append(field.DefaultFields, confschema.Fields...)
+	// Ensure unique fields
+	uniqueFields := make(map[string]field.SchemaField)
+	for _, f := range confschema.Fields {
+		uniqueFields[f.FieldName] = f
```
Comment (pquerna) [inline pkg/config/config.go:51]: should this panic or otherwie fail?

Comment (btipling) [reply pkg/config/config.go:51]: It should panic these are schema definitions we wrote

Comment (btipling) [reply pkg/config/config.go:51]: It fails later when we attempt to add a duplicate flag actually

Comment (shackra) [reply pkg/config/config.go:51]: why should this panic tho?

Comment (pquerna) [reply pkg/config/config.go:51]: duplicate flags was my worry

**Pull request merged**